### PR TITLE
#389: MapException support in Modeler

### PR DESCRIPTION
**Pull request by Blikingor**
Updates to converters, UI elements and unit tests to support flowable:mapException for ServiceTasks in the modeler.

### CODE REVIEW

File: modules/flowable-bpmn-converter/src/main/java/org/flowable/bpmn/converter/ServiceTaskXMLConverter.java [hunk 1]
```diff
@@ -12,20 +12,14 @@
...
 import org.flowable.bpmn.converter.util.BpmnXMLUtil;
-import org.flowable.bpmn.model.AbstractFlowableHttpHandler;
-import org.flowable.bpmn.model.BaseElement;
-import org.flowable.bpmn.model.BpmnModel;
-import org.flowable.bpmn.model.CaseServiceTask;
-import org.flowable.bpmn.model.CustomProperty;
-import org.flowable.bpmn.model.HttpServiceTask;
-import org.flowable.bpmn.model.ImplementationType;
-import org.flowable.bpmn.model.ServiceTask;
+import org.flowable.bpmn.model.*;
```
Comment (dbmalkovsky) [inline modules/flowable-bpmn-converter/src/main/java/org/flowable/bpmn/converter/ServiceTaskXMLConverter.java]: Please note the project does not use wildcard imports but explicitly lists each import.  Thanks.

**Comment by PascalSchumacher** [comment]
Travis failure on jdk 12 seems unrelated to the changes in this pull request:

>TerminateEndEventTest.testTerminateInSubProcessWithBoundary:385->AbstractFlowableTestCase.assertProcessEnded:174->AbstractFlowableTestCase.assertProcessEnded:181 Expected finished process instance '72683' but it was still in the db

**Comment by Blikingor** [comment]
Hmm, not sure what I can do to make the tests pass. Any ideas @PascalSchumacher ?

File: modules/flowable-json-converter/src/test/java/org/flowable/editor/language/ServiceTaskConverterTest.java [hunk 2]
```diff
@@ -12,20 +12,19 @@
...
 import org.flowable.bpmn.model.FlowElement;
+import org.flowable.bpmn.model.FlowableListener;
 import org.flowable.bpmn.model.ImplementationType;
+import org.flowable.bpmn.model.MapExceptionEntry;
 import org.flowable.bpmn.model.ServiceTask;
 import org.junit.Test;
 
+import java.util.List;
+
+import static org.junit.Assert.*;
```
Comment (dbmalkovsky) [inline modules/flowable-json-converter/src/test/java/org/flowable/editor/language/ServiceTaskConverterTest.java]: Looks like most wildcard imports have been corrected but this one at least was missed.
BTW, standards have static imports at the top of the file not at the bottom of the list.

Comment (Blikingor) [reply modules/flowable-json-converter/src/test/java/org/flowable/editor/language/ServiceTaskConverterTest.java]: Fix on the way

**Comment by Blikingor** [comment]
Any word on getting this one merged?

**Comment by tijsrademakers** [comment]
Sorry for the delay, we still need to do some testing. I expect this can be done before the end of next week.

**Comment by tijsrademakers** [comment]
Thanks for PR!

**Pull request merged**